src/pages/home/report/ReportActionCompose.js

```diff
@@ -168,24 +181,35 @@ class ReportActionCompose extends React.Component {
                             <>
                                 <AttachmentPicker>
                                     {({openPicker}) => (
-                                        <TouchableOpacity
-                                            onPress={(e) => {
-                                                e.preventDefault();
-
-                                                // Do not open attachment picker from keypress event
-                                                if (!e.key) {
-                                                    openPicker({
-                                                        onPicked: (file) => {
-                                                            displayFileInModal({file});
-                                                        },
-                                                    });
-                                                }
-                                            }}
-                                            style={[styles.chatItemAttachButton]}
-                                            underlayColor={themeColors.componentBG}
-                                        >
-                                            <Icon src={Paperclip} />
-                                        </TouchableOpacity>
+                                        <>
+                                            <TouchableOpacity
+                                                onPress={(e) => {
+                                                    e.preventDefault();
+                                                    this.setMenuVisibility(true);
+                                                }}
+                                                style={styles.chatItemAttachButton}
+                                                underlayColor={themeColors.componentBG}
+                                            >
+                                                <Icon src={Paperclip} />
```

Comment:
left this icon as a `Paperclip`, might want to update to make it seem more like a menu icon

Reply:
Ah good point, I forgot to mention but we have a new icon for this.